data/build.gradle.kts

```
@@ -0,0 +1,25 @@
+@file:Suppress("DSL_SCOPE_VIOLATION", "UnstableApiUsage")
+
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.dagger.hilt)
+    alias(libs.plugins.kotlin.kapt)
+}
+
+android {
+    compileSdk = 33
+    namespace = "com.yapp.data"
+    kotlinOptions {
+        jvmTarget = "11"
```

```suggestion
        jvmTarget = JavaVersion.VERSION_11.majorVersion
```
내부코드를 보니 이렇게도 적을 수 있을 것 같아서 참고차 코멘트 답니당....

---

toml에 관리하는 것 보다 제공하는 상수를 사용하는게 나을 것 같네 고마워!